frontend/app/src/locales/gr.json

```diff
@@ -626,6 +626,25 @@
         "message_title": "Backend messages"
       }
     },
+    "online_events": {
+      "error": {
+        "description": "Αποτυχία ερωτήματος online εκδηλώσεων με τύπο ερωτήματος {queryType}: {error}",
+        "title": "Ερώτημα Εκδηλώσεων Σε Απευθείας Σύνδεση"
```

Comment:
"Εκδηλώσεων" this is the wrong word to use here. This is used to describe the type of events that people attend.

Let me check the English text and update the translation.